frontend/components/auction/collateral/CollateralAuctionFlow.vue

```diff
@@ -28,7 +28,7 @@
                         :take-events="selectedTakeEvents"
                         :wallet-address="walletAddress"
                         :auction-id="selectedAuctionId"
-                        :are-auctions-fetching="areAuctionsFetching || isSelectedAuctionFetching"
+                        :are-auctions-fetching="areAuctionsFetching"
```

Comment:
Why was this removed? When all auctions are not fetched (but only a single auction is) `areAuctionsFetching` will not be set to my knowledge

Reply:
But that was intended to have separate loading indicators. The reason is to not mislead user by showing loading on the table with auctions when in actuality we only fetch a single auction